tools/sof-dump-status.py

```diff
@@ -527,6 +557,13 @@ def dump_dapm(dapm, filter = "all"):
     # key is used. Here we output firmware path according to kernel's
     # match table.
     if ret_args.get('fwpath') is True:
+        sysinfo.loadFw()
+
+        if len(sysinfo.fw_lst) != 0:
+            fw_path = "/lib/firmware/imx/sof"
```

Comment:
So `/sys/devices/soc0/soc_id` is IMX-only? Then it should be much clearer in comments and `imx` should be somewhere in the list name.